README.md

```diff
@@ -10,9 +18,9 @@ This Cloudformation and accompanying Lambda function seek to create a one-way in
 
 ## Deployment
 
-1. Configure the Datadog Event Bridge Integration.  Documentation exists for this [here](https://docs.datadoghq.com/integrations/amazon_event_bridge/).
-2. Create a notification rule naming @awseventbridge-YOUR_BRIDGE_NAME as the destination
-3. Deploy the SAM template.  You'll need to provide the EventBridge bus name and the ARN of your SecurityHub.
+1. Configure the Datadog Event Bridge Integration. Documentation exists for this [here](https://docs.datadoghq.com/integrations/amazon_event_bridge/).
+2. In Datadog, create a [security notification rule](https://docs.datadoghq.com/security_platform/notification_rules/) naming @awseventbridge-YOUR_BRIDGE_NAME as the destination.
+3. Deploy SAM template.  You'll need to provide the EventBridge bus name and the ARN of your SecurityHub.
```

Comment:
Can you change line 23 to say "Deploy the SAM template" to align with our other docs